sources/packages/backend/libs/integrations/src/object-storage/object-storage.service.ts

```diff
@@ -32,11 +33,12 @@ export class ObjectStorageService {
   /**
    * Saves an object to the storage.
    * @param object object details to be stored.
+   * @returns httpStatusCode http status code.
    * @throws S3ServiceException base exception class for all service exceptions from S3 service.
    * @see https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/s3/command/PutObjectCommand/
    */
-  async putObject(object: StorageObject): Promise<void> {
-    await this.s3Client.send(
+  async putObject(object: StorageObject): Promise<number> {
```

Comment:
This change can be reverted as per comment https://github.com/bcgov/SIMS/pull/3663#discussion_r1747489606